seqr/migrations/0055_alter_sample_tissue_type.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 3.2.18 on 2023-09-28 16:05
+
+from collections import defaultdict
+from django.db import migrations, models
+from tqdm import tqdm
+
+SAMPLE_TISSUE_MAP = {sample_id: 'A' for sample_id in {
+    'MCH-103_1_NE1', 'MCH-75_1_NE1', 'MCH-44_1_NE1', 'MCH-80_1_NE1', 'MCH-199_1_NE1', 'MCH-68_1_NE1', 'MCH-55_1_NE1',
+    'MCH-69_1_NE1', 'MCH-198_1_NE1',
+}}
+
+
+def update_tissue_type(apps, schema_editor):
+    Sample = apps.get_model('seqr', 'Sample')
+    rna_data_classes = {
+        'rnaseqoutlier': apps.get_model('seqr', 'RnaSeqOutlier'),
+        'rnaseqspliceoutlier': apps.get_model('seqr', 'RnaSeqSpliceOutlier'),
+        'rnaseqtpm': apps.get_model('seqr', 'RnaSeqTpm'),
+    }
+    db_alias = schema_editor.connection.alias
+
+    samples = Sample.objects.using(db_alias).filter(tissue_type__isnull=True)
+    if not samples:
+        return
+
+    rna_samples = samples.filter(sample_type='RNA', is_active=True)
+    non_rna_samples = samples.exclude(id__in=[s.id for s in rna_samples])
```

Comment:
no because its AND vs OR logic. The equivalent would be `filter(~Q(sample_type='RNA') | Q(is_active=False))` which felt less readable but I can change to if you think its more explicit

Reply:
cool, I wouldn't bother changing it since this is a migration.  That makes sense to me though.